substrate/frame/assets/src/impl_sufficiency.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{
```

Comment:
maybe `traits` and `impls`? otherwise the list of files will grow here pretty quick. 
I also have only strong opinion on keeping our folders/files/modules style consistent, since there is no yet agreement on it, I would just try to keep it close to what we have now, and now we tend to have less files than many.

Reply:
Would this mean also moving `impl_fungibles` and `impl_stored_map` onto an `impls` file/folder? I see the point on making the file structure lighter, but seeing that other impls are not that light (i.e. `impl_fungibles` is 311 lines) a bit of _not-flatness_ might be good for the sake of readibility from time to time.

My suggestion:

```
impls/
├─ fungibles.rs
├─ stored_maps.rs
├─ sufficiency.rs
├─ mod.rs
traits.rs
```